File: Daily_Codes_Answers/day-20/2974_Minimum_Number_Game.java
import java.util.*;
class Solution {
    static int[] numberGame(int[] nums) {
        Arrays.sort(nums);
        int arr[] = new int[nums.length];
        for (int i=0 ; i<nums.length; i+=2){
            arr[i] = nums[i+1];
            arr[i+1] = nums[i];
        }
        return arr;
    }
    public static void main(String[] args) {
        int arr1[] = new int[]{5,4,2,3};
        int arr2[] = new int[]{2,5};

        System.out.println("-- test case 1 --");
        System.out.println("Input : "+Arrays.toString(arr1)+"\nOutput : "+Arrays.toString(numberGame(arr1)));
        System.out.println("-- test case 2 --");
        System.out.println("Input : "+Arrays.toString(arr2)+"\nOutput : "+Arrays.toString(numberGame(arr2)));
    }
}

File: Daily_Codes_Answers/day-20/2974_Minimum_Number_Game.js
const numberGame = (nums)=>{
    let arr = []
    nums.sort()
    for (let i =0 ; i<nums.length ; i+=2){
        arr.push(nums[i+1])
        arr.push(nums[i])
    }
    return arr
}

let arr1 = [5,4,2,3]
let arr2 = [2,5]

console.log("-- test case 1 --");
console.log("Input : "+arr1+"\nOutput : "+numberGame(arr1))
console.log("-- test case 2 --");
console.log("Input : "+arr2+"\nOutput : "+numberGame(arr2))